<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WVoicelibWidgetClass</class>
 <widget class="QWidget" name="WVoicelibWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>153</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>153</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>153</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WVoicelibWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QFrame#frame{
	border:1px solid #C9C9C9;
	border-radius:6px;
}

QFrame#frame_2{
	border:1px;
	border-bottom:1px solid #C9C9C9;
}

QLabel{
	border:none;
	background-position: center;
    background-repeat: no-repeat;
}

QLabel#lb_content,#lb_num{
	color: #848484;
}

QLabel#lb_1[man=true]{
	background-image: url(:/QtTest/icon/Setting/soundbot/man.png);
}

QLabel#lb_1[man=false]{
	background-image: url(:/QtTest/icon/Setting/soundbot/waman.png);
}

QLabel#lb_2{
	background-position: top;
	background-image: url(:/QtTest/icon/Setting/soundbot/little.png);	
}

QLabel#lb_4{
	background-image: url(:/QtTest/icon/Setting/soundbot/peo.png);	
}

QPushButton#pb_sample{
	border:none;
	border-radius:6px;
	padding-right:18px;
	background-origin: content;
    background-position: left;
    background-repeat: no-repeat;
	background-image: url(:/QtTest/icon/Setting/soundbot/btn_1.png);
	background-color: ;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);

	text-align:right;
	padding-left:28px;
	color:#ffffff;
}

QPushButton#pb_add{
	border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
	border-radius:6px;
	padding-right:5px;
	background-origin: content;
    background-position: left;
    background-repeat: no-repeat;
	background-color: #ffffff;
	background-image: url(:/QtTest/icon/Setting/soundbot/btn_2.png);

	text-align:right;
	padding-left:10px;
	color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);;
}









</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>289</width>
       <height>152</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>289</width>
       <height>152</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="rightMargin">
            <number>6</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="lb_1">
               <property name="minimumSize">
                <size>
                 <width>15</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>15</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>9</width>
                 <height>16</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lb_name">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>18</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>18</height>
              </size>
             </property>
             <property name="text">
              <string>General translation</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="lb_4">
               <property name="minimumSize">
                <size>
                 <width>11</width>
                 <height>11</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>11</width>
                 <height>11</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_num">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
               <property name="text">
                <string>123532</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lb_content">
           <property name="text">
            <string>This voice has been randomly generated</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="pb_sample">
          <property name="minimumSize">
           <size>
            <width>109</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>109</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Sample</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_add">
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>133</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Add to my Voice</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../AiSound.qrc"/>
 </resources>
 <connections/>
</ui>
